config/application.rb

```diff
@@ -2,18 +2,12 @@
 
 require 'rails/all'
 
-# Require the gems listed in Gemfile, including any gems
-# you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
 module TestGuru
   class Application < Rails::Application
-    # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
-
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration can go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.
+    config.time_zone = 'Ekaterinburg'
+    config.i18n.default_locale = 'ru-RU'
```

Comment:
```suggestion
    config.i18n.default_locale = :ru
```
этого достаточно
[документация](https://guides.rubyonrails.org/i18n.html#configure-the-i18n-module) по конфигурированию "дефолтной локали" в рельсах
обрати внимание: основной вариант работы с локалью включает только указания языка, регион не указывается